README.md

```diff
@@ -1,147 +1,184 @@
 
-```bash
-cl /?
-link /?
-```
+  - It is recommended to use MiniForge instead of Powershell as the main terminal to build and write code for LPython.
 
-Both commands must print several pages of help text.
+- In the MiniForge Prompt, initialize the MSVC compiler using the below command:
 
-## Build LPython
+  ```bash
+  call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd" -arch=x64
+  ```
 
-Clone LPython
+- You can optionally test MSVC via:
 
-```bash
-git clone https://github.com/lcompilers/lpython.git
-cd lpython
-```
+  ```bash
+  cl /?
+  link /?
+  ```
+
+  Both commands must print several pages of help text.
+
+## Step 3: Build LPython
+
+- Clone LPython using the following commands
+
+  ```bash
+  git clone https://github.com/lcompilers/lpython.git
+  cd lpython
+  ```
+
+  You may also use GitHub Desktop to do the same.
 
 ### Linux and MacOS
 
 - Create a Conda environment using the pre-existing file:
 
-```bash
-conda env create -f environment_unix.yml
-conda activate lp
-```
+  ```bash
+  conda env create -f environment_unix.yml
+  conda activate lp
+  ```
 
 - Generate prerequisite files; build in Debug Mode:
 
-```bash
-./build0.sh
-./build1.sh
-```
+  ```bash
+  ./build0.sh
+  ./build1.sh
+  ```
 
 ### Windows
 
 - Create a Conda environment using the pre-existing file:
 
-```bash
-conda env create -f environment_win.yml
-conda activate lp
-```
+  ```bash
+  conda env create -f environment_win.yml
+  conda activate lp
+  ```
 
 - Generate prerequisite files; build in Release Mode:
 
-```bash
-call build0.bat
-call build1.bat
-```
+  ```bash
+  call build0.bat
+  call build1.bat
+  ```
 
-- Tests and examples
+- Tests and examples (Optional)
 
-```bash
-ctest
-inst\bin\lpython examples\expr2.py
-inst\bin\lpython examples\expr2.py -o a.out
-a.out
-```
+  ```bash
+  ctest
+  inst\bin\lpython examples\expr2.py
+  inst\bin\lpython examples\expr2.py -o a.out
+  a.out
+  ```
 
-- After you update a test case file, you also need to update all the reference results associated with that test case:   
+- Whenever you are updating a test case file, you also need to update all the reference results associated with that test case:
 
-```
-python run_tests.py -u --skip-run-with-dbg
-```
+  ```
+  python run_tests.py -u --skip-run-with-dbg
+  ```
 
 - To see all the options associated with LPython test suite, use:
-```
-python run_tests.py --help
-```
 
-## Tests (Linux or MacOs):
+  ```
+  python run_tests.py --help
+  ```
 
-Run tests:
+## Tests:
 
-```bash
-ctest
-./run_tests.py
-```
+### Linux or MacOS
 
-Run integration tests:
+- Run tests:
 
-```bash
-cd integration_tests
-./run_tests.py
-```
+  ```bash
+  ctest
+  ./run_tests.py
+  ```
+
+- Run integration tests:
+
+  ```bash
+  cd integration_tests
+  ./run_tests.py
+  ```
 
-### Speed up Integration Test on Macs
+### Windows
+
+- Run integration tests
+
+  ```bash
+  cd integration_tests
+  python run_tests.py
+  ```
```

Comment:
I think this doesn't work in Windows.

Reply:
Instead, this works: `python run_tests.py --skip-run-with-dbg`

Reply:
Will change the command